lib/core/src/irods_server_properties.cpp

```diff
@@ -214,13 +214,17 @@ namespace irods
         std::string svr_fn;
         irods::error ret = irods::get_full_path_for_config_file(SERVER_CONFIG_FILE, svr_fn);
         if (!ret.ok()) {
-            return;
+            THROW(ret.code(),
+                  fmt::format("[{}:{}] - Failed to find server config file [{}]", __func__, __LINE__, ret.result()));
         }
 
         std::ifstream svr{svr_fn};
-        if (svr.is_open()) {
-            config_props_ = json::parse(svr);
+
+        if (!svr || !svr.is_open()) {
```

Comment:
`!svr.is_open()` is not needed because `!svr` covers that case.

`!svr` is basically asking if the stream is in a bad state.

Reply:
Oh, cool. Shows how much I know about the streaming library :p

Reply:
Done